app/src/common/shared/com/igalia/wolvic/ui/widgets/TrayWidget.java

```diff
@@ -509,7 +509,6 @@ protected void initializeWidgetPlacement(WidgetPlacement aPlacement) {
         aPlacement.rotationAxisX = 1.0f;
         aPlacement.rotation = (float)Math.toRadians(-45);
         aPlacement.cylinder = false;
-        aPlacement.textureScale = 1.0f;
```

Comment:
We need to figure out how to remove the gaps when textureScale is lower than 1.0f instead of simply fix the textureScale to 1 here https://github.com/MozillaReality/FirefoxReality/pull/790#issuecomment-440792219

Reply:
Fixed by https://github.com/Igalia/wolvic/pull/1086/files#diff-bff0a507df2c8235d7a28517e9aedb6e8b49af6565b9f6873d8e60d2e289cd7bR150
(hacky but works)